.github/workflows/node.js.yml

```diff
@@ -146,6 +146,7 @@ jobs:
           NEXTAUTH_SECRET: ${{ secrets.NEXTAUTH_SECRET }}
           GITHUB_CLIENT_ID: ${{ secrets.AUTH_GITHUB_CLIENT_ID }}
           GITHUB_CLIENT_SECRET: ${{ secrets.AUTH_GITHUB_CLIENT_SECRET }}
+          PAYPAL_CLIENT_SECRET: ${{ secrets.PAYPAL_CLIENT_SECRET }}
```

Comment:
We need `NEXT_PUBLIC_PAYPAL_CLIENT_ID` and `PAYPAL_URL` as well. Otherwise, both envs would be undefined on prod.